app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id] unless @user
```

Comment:
check đk như này ko hợp lý e nhé, hiện tại đang tìm @user nếu không có @user, thế ban đầu @user lấy đâu ra để check vậy em?
```suggestion
    @user = User.find_by id: params[:id]
    unless @user
      xử lý không tìm thấy ở đây
    end
```
hoặc
```suggestion
    @user = User.find_by id: params[:id]
    return if @user
    xử lý không có @user ở đây
```